src/js/Controllers/UIController.js

```diff
@@ -200,9 +200,9 @@ class UIController {
                     rpc.params.cancelID,
                     rpc.params.timeout
                 ))
-                var timeout = rpc.params.timeout === 0 ? 15000 : rpc.params.timeout
+                const timeout = rpc.params.timeout === 0 ? 15000 : rpc.params.timeout
```

Comment:
@LiliiaShlikhtLuxoft Where is `15000` come from?Is somewhere requirement for this value? Maybe default const?

Reply:
Done